pkg/model/multiversion/api_info.go

```diff
@@ -0,0 +1,40 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package multiversion
+
+// TODO(a-hilaly) move this file outside of pkg/model/multiversion. Idealy we
+// Should be able to access APIStatus and APIInfo to prevent regenerating removed or
+// deprecated APIs.
+// TODO(a-hilaly) store API status in `ack-generate-metadata.yaml`
```

Comment:
nit: As part of discussions regarding the metadata config file, not true anymore. `metadata.yaml` will be source of truth for API statuses

Reply:
Yup ++, removing the TODO